mantidproject/mantid
# Conda build fixes

**Description of work.**
Some upstream fixes are required to correctly build and packages mantidworkbench with the docs and required QT plugins for launching mantidworkbench.



**To test:**
- Use the packages made by Jenkins will be linked below, check that the packages open, this is not a conda package build, we need to test the normal packaging, and that the normal packages launch properly.
- [x] Ubuntu: [Package](https://builds.mantidproject.org/job/pull_requests-ubuntu/49805/artifact/build/mantidunstable_6.2.20220117.950-0ubuntu1%7Ebionic1_amd64.deb) @thomashampson 
- [x] RHEL7: [Package](https://builds.mantidproject.org/job/pull_requests-rhel7/48757/artifact/build/mantidunstable-6.2.20220117.901-1.el7.x86_64.rpm) @Pasarus
- [x] MacOSX: [Package](https://builds.mantidproject.org/job/pull_requests-osx/48193/artifact/build/mantidunstable-6.2.20220117.901-HighSierra.dmg) @DanielMurphy22 
- [x] Windows: [Package](https://builds.mantidproject.org/job/pull_requests-windows/48603/artifact/build/mantidunstable-6.2.20220117.901-win64.exe) @DanielMurphy22 


*This does not require release notes* because **packaging related**




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
